style.css

```diff
@@ -0,0 +1,137 @@
+.toolbar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background: #3c3a39;
+}
+
+.toolbar:hover {
+  border: 0.25px solid red;
+  outline: 0.5px solid blue;
+}
+
+.name {
+  display: flex;
+  flex-grow: 1;
+  flex-shrink: 0;
+  left: 16px;
+  order: 0;
+  margin: 0 0;
+  position: absolute;
+  width: 500px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.menu {
+  position: absolute;
+  width: 20px;
+  right: 16px;
+  background: #fff;
+}
+
+.page1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background: #1c1a19;
+}
```

Comment:
- [ ] Please make the `section` and the `header` responsive with the mobile size as they have to be in the Figma design suggestions you can remove display: absolute and make the width of the `header` and the `section` 100%.

your design 
![Screenshot from 2022-04-28 00-32-48](https://user-images.githubusercontent.com/33752735/165642259-33027608-a526-48aa-933c-cc6110b5e806.png)

Figma design
![Screenshot from 2022-04-27 20-47-24](https://user-images.githubusercontent.com/33752735/165642331-1d3789b6-9202-428a-9ea0-b53d3bb96664.png)